corehq/util/context_processors.py

```diff
@@ -266,3 +266,9 @@ def get_demo(request):
             'is_demo_visible': True,
         })
     return context
+
+
+def status_page(request):
+    return{
+        'show_status_page': settings.SERVER_ENVIRONMENT == 'production'
```

Comment:
If you also want to `show_status_page` to be true for both `production` and `staging` then you can use `settings.IS_SAAS_ENVIRONMENT` which basically tests the same thing. 

```suggestion
        'show_status_page': settings.IS_SAAS_ENVIRONMENT
```

Reply:
This is a really good idea